ossindex/ossindex.go

```diff
@@ -43,6 +44,12 @@ var (
 	ossIndexUrl string
 )
 
+// func ClearCache() {
+// 	db, err := openDb(getDatabaseDirectory())
+// 	customerrors.Check(err, "Error opening database")
+// 	defer db.Close()
+// }
+
```

Comment:
We should kill this if we dont plan on bringing it back.

Reply:
In: 2e0f0fc